doc/dune

```diff
@@ -1,5 +1,6 @@
 (rule
  (targets unreleased.rst)
+ (mode promote-until-clean)
```

Comment:
I see the following in the log of the failed refman build, and it could be related to this change:
```
File "config/dune", line 15, characters 0-402:
15 | (rule
16 |  (targets coq_config.ml coq_config.py Makefile dune.c_flags)
17 |  (mode fallback)
18 |  (deps
19 |    %{project_root}/configure.ml
20 |    %{project_root}/dev/ocamldebug-coq.run
21 |    %{project_root}/dev/header.c
22 |    ; Needed to generate include lists for coq_makefile
23 |    plugin_list
24 |   (env_var COQ_CONFIGURE_PREFIX))
25 |  (action (chdir %{project_root} (run %{ocaml} configure.ml -no-ask -native-compiler no -bin-annot))))
Error: Some of the targets of this fallback rule are present in the source
tree, and some are not. This is not allowed. Either none of the targets must
be present in the source tree, either they must all be.
The following targets are present:
- config/Makefile
- config/coq_config.ml
- config/coq_config.py
The following targets are not:
- config/dune.c_flags
```

Reply:
Let me have a look, this is a new problem with the last version of the PR.